freqtrade/rpc/api_server/webserver.py

```diff
@@ -127,10 +128,10 @@ def shutdown(cls):
         cls._has_rpc = False
         cls._rpc = None
 
-    def send_msg(self, msg: Dict[str, str]) -> None:
+    def send_msg(self, msg: Dict[str, Any]) -> None:
         if self._ws_queue:
             sync_q = self._ws_queue.sync_q
-            sync_q.put(msg)
+            sync_q.put(WSMessageSchema(**msg))
```

Comment:
we assume this is cheap, right?

I'll run some more tests with this - as i don't trust pydantic to be fast in all instances (especially if the object becomes bigger)...

Reply:
I believe it's either this or `parse_obj` and iirc they both are the same except `parse_obj` will ignore unknown fields

Reply:
yes, the "how" is not the point, if any, it's the "if"...
the alternative i see would be to revert to prior behavior, passing in a dict to the send_direct message (not a WSMessageSchema object) - as i've clearly missed that we're currently not doing this transformation for broadcasts.

Reply:
I'm not sure I follow, this line is the transformation of dict to WSMessageSchema for broadcast

Reply:
I've decided to revert this to be a dict. It'll avoid type-checking outgoing messages, where we know (or at least assume we know) what the content is. As we expect this to potentially be called 200 (once per pair) times per iteration (worst case) - this can have a noticable impact.